ajax/libs/jquery.printarea/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "jquery.printarea",
+  "version": "2.4.1",
+  "description": "Prints a specific area of the page. Using json settings, the printed area is either opened in a popup or in a hidden iframe. Minimal setup and easy to use. See the demo for examples and usage of the options.",
+  "keywords": [
+    "jquery",
+    "print",
+    "printing"
+  ],
+  "filename": "jquery.PrintArea.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RitsC/PrintArea.git"
+  },
+  "authors": [
+    "Chris Ritschard",
+    "Djuri Baars"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/RitsC/PrintArea.git",
+    "fileMap": [
+      {
+        "basePath": "demo",
+        "files": [
+          "jquery.PrintArea*"
```

Comment:
Ummm I think use `demo`as the basePath is a little weird.
What do you think?

@extend1994 @maruilian11 Could you give some suggestion about this issue?

Many thanks.

Reply:
This is author build folder... I can change it to regular css and js and minifiy them. However, for you suggestion.
I found that pattern in autoupdate was wrong. Was updated. FYR